.rubocop.yml

```diff
@@ -26,3 +26,9 @@ RSpec/Focus:
 
 RSpec/MultipleExpectations:
   Enabled: false
+
+Style/TrailingCommaInArrayLiteral:
+  EnforcedStyleForMultiline: consistent_comma
+
+Style/TrailingCommaInHashLiteral:
```

Comment:
Without this rubocop was insisting that the trailing comma in `BROKEN_CONFIG_VARS` be removed. Rather than just enabling trailing commas for hashes, I've enabled them for arrays too, for consistency.